setup.py

```diff
@@ -62,8 +63,4 @@ def get_requirements():
     install_requires=get_requirements(),
     scripts=['bin/dk_monitor'],
     test_suite='nose.collector',
-    entry_points="""
```

Comment:
So was this stuff that we needed for rake? Or just cruft?

Reply:
It was cruft